### DISCUSSION TITLE
Welcome to Pundit discussions!

### DISCUSSION THREAD

**Discussion by Burgestrand** [varvet/pundit#698] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jasper502** [comment]
Was turning off new discussions intentional?

**Reply by Burgestrand** [reply]
New discussions should be possible in all categories except this one, `Announcements`! This one is reserved for maintainers and admins of the repository so that we have a low-noise space for announcements 😄

**Reply by jasper502** [reply]
Hmmm... something is off then.  I don't see the 'New discussion' button in the other groups.  I can see this in some other repositories here is what I see in the Q&A section and the same from another repository I am active in.
![Screen Shot 2022-05-08 at 7 19 45 PM](https://user-images.githubusercontent.com/1505829/167325123-8416fcbf-f4c1-4323-9255-a48f6ad65503.png)
![Screen Shot 2022-05-08 at 7 20 00 PM](https://user-images.githubusercontent.com/1505829/167325119-7f7e7628-d666-4fe6-b8d6-6062ad1f5d83.png)

**Reply by tylerpaige** [reply]
Like @jasper502, I am also unable to create a new discussion

**Reply by laptopmutia** [reply]
I cannot create a new discussions

**Reply by Burgestrand** [reply]
It took me a while to find the setting, but the github organization had discussions limited to users with at least triage-level access.

I created a new github account to verify, and indeed it does have the "New discussion" button visible now. Hopefully this is now fixed!
